src/Bridge/Symfony/Blacklister/Not2xxBlacklister.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the SeoOverride project.
+ *
+ * (c) JoliCode <coucou@jolicode.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Joli\SeoOverride\Bridge\Symfony\Blacklister;
+
+use Joli\SeoOverride\Bridge\Symfony\Blacklister;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class Not2xxBlacklister implements Blacklister
+{
+    public function isBlacklisted(Request $request, Response $response): bool
+    {
+        return $response->getStatusCode() >= 300;
```

Comment:
This code does not do what the fixer say it does (what about 1xx codes?)

Reply:
Should we really care about 1xx responses? Not sure a PHP app can/should return such statuses code